packages/clerk-js/src/ui/elements/Card/CardAction.tsx

```diff
@@ -2,23 +2,27 @@ import type { CardActionId } from '@clerk/types';
 import React from 'react';
 
 import { descriptors, Flex, Text } from '../../customizables';
-import type { PropsOfComponent } from '../../styledSystem';
+import type { PropsOfComponent, ThemableCssProp } from '../../styledSystem';
 import { RouterLink } from '../RouterLink';
 
 type CardActionProps = Omit<PropsOfComponent<typeof Flex>, 'elementId'> & {
   elementId?: CardActionId;
+  sx?: ThemableCssProp;
```

Comment:
Isn't `sx` already included in `PropsOfComponent<typeof Flex>` ?

Reply:
Good catch, removed!